pkg/backup/testing.go

```diff
@@ -12,7 +11,7 @@ import (
 func createControlDatabase(t *testing.T) DB {
 	t.Helper()
 
-	f, err := ioutil.TempFile(t.TempDir(), "control_*.db")
+	f, err := os.CreateTemp(t.TempDir(), "control_*.db")
```

Comment:
Had some lint warnings about this here and similar in a couple other places.

Reply:
yeah. `ioutil` pkg is deprecated. I tried to find the equivalent in the `os` but did not find it at the time. are you using a different linter or how did you set it up? I did not see any warnings

Reply:
You know, I'm not sure how my vscode linter is picking up these extra things that `make lint` isn't. Could be part of vscode running gofumpt for formatting, or the fact that i'm using go 1.19 now? Will have to research this more to make sure my vscode and `make lint` do the same thing, and that we all get those settings coming along with the repo.

Reply:
So, this one is showing up for me when running `make lint`.

```
> make lint
go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2 run
pkg/backup/testing.go:4:2: SA1019: package io/ioutil is deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
exit status 1
make: *** [lint] Error 1
```

Do you not see that? I am running Go 1.19.